src/styles/block/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $white;
+  padding: 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    &--modifier {
+      margin: 16px auto 0;
+    }
+  }
+
+  &__title {
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    &--modifier {
+      margin-top: 40px;
+      margin-bottom: 0;
+    }
+  }
+
+  &__code {
+    color: $other-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    &--modifier {
+      margin-top: 4px;
+    }
+  }
+
+  &__review {
+    @include addSpaceBetween;
+
+    &--modifier {
+      margin-top: 16px;
+    }
+
+    &-count {
+      display: flex;
+      height: 16px;
+      justify-content: flex-end;
+      align-items: center;
+      color: $main-text-color;
+      padding: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__prise {
+    @include addSpaceBetween;
+    height: 18px;
+
+    &--modifier {
+      margin-top: 24px;
+    }
+
+    &-title {
+      color: $other-text-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-baks {
+      color: $main-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: $white;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    width: 166px;
```

Comment:
Setting `width` for the button here is not perfect. Better let it occupy the full width of the parent. In addition - `width: 166px` has no effect here